Assets/MixedRealityToolkit/UX/Scripts/AppBar/AppBar.cs

```diff
@@ -9,7 +9,9 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.Events;
-using UnityEngine.XR.WSA.Input;
+
+
+//using UnityEngine.XR.WSA.Input;
```

Comment:
Don't leave commented out code in the file. Just delete it. (git is your friend)